priv/repo/migrations/20200408144038_remove_respondent_stalled_state.exs

```diff
@@ -0,0 +1,54 @@
+defmodule Ask.Repo.Migrations.RemoveRespondentStalledState do
+  use Ecto.Migration
+  import Ecto.Query
+  import Ecto
+  alias Ask.Repo
+
+  defmodule Respondent do
+    use Ask.Web, :model
+
+    schema "respondents" do
+      field :session, Ask.Ecto.Type.JSON
+      field :state, :string, default: "pending"
+      field :timeout_at, Timex.Ecto.DateTime
+      field :disposition, :string, default: "registered"
+    end
+
+    def update_stalled_to_failed(respondent) do
+      new_disposition = failed_disposition_from(respondent.disposition)
+
+      respondent
+      |> Respondent.changeset(%{state: "failed", session: nil, timeout_at: nil, disposition: new_disposition})
+      |> Repo.update!
```

Comment:
Won't we miss a `RespondentDispositionHistory` here?
 https://github.com/instedd/surveda/blob/162e501be543833cf2186cbb49c6dab4aa104ad3/lib/ask/runtime/survey.ex#L130